sponsors/serializers.py

```diff
@@ -0,0 +1,82 @@
+
+from rest_framework import serializers
+
+from sponsors.models import GenericAsset
+from sponsors.models.enums import PublisherChoices, LogoPlacementChoices
+
+class LogoPlacementSerializer(serializers.Serializer):
+    publisher = serializers.CharField()
+    flight = serializers.CharField()
+    sponsor = serializers.CharField()
+    sponsor_slug = serializers.CharField()
+    description = serializers.CharField()
+    logo = serializers.URLField()
+    start_date = serializers.DateField()
+    end_date = serializers.DateField()
+    sponsor_url = serializers.URLField()
+    level_name = serializers.CharField()
+    level_order = serializers.IntegerField()
+
+
+class AssetSerializer(serializers.ModelSerializer):
+    content_type = serializers.SerializerMethodField()
+    value = serializers.SerializerMethodField()
+    sponsor = serializers.SerializerMethodField()
```

Comment:
can we add the sponsor slug to this as well? I think we're all set then!

Reply:
Done in ebc2227e